### DISCUSSION TITLE
(dropbox) 64-bit desktop app version available now

### DISCUSSION THREAD

**Discussion by henkisdabro** [chocolatey-community/chocolatey-packages#2004] [Ideas]
According to this article, there is now (finally) a 64-bit version of Dropbox available.
https://www.dropboxforum.com/t5/Apps-and-Installations/Can-64-bit-Dropbox-be-downloaded-on-Windows/td-p/543492

**Comment by RedBaron2** [comment]
@henkisdabro 
Would you like to submit the PR with the required changes to implement this change?

If not I could submit the PR later this week.

**Reply by henkisdabro** [reply]
I would love to help, but don't have the required skills to do so (yet!), I'm a mere humble user :-)

**Reply by RedBaron2** [reply]
@henkisdabro 
I'll see about submitting the PR this weekend.

**Comment by RedBaron2** [comment]
@henkisdabro 
The PR #2019  is up for review now
